Makefile

```diff
@@ -203,6 +204,7 @@ $(libceed) : LDFLAGS += $(if $(DARWIN), -install_name @rpath/$(notdir $(libceed)
 libceed.c += $(ref.c)
 libceed.c += $(template.c)
 libceed.c += $(blocked.c)
+libceed.c += $(avx.c)
```

Comment:
This will only build if we're on a system that supports AVX. On Linux, we can look at `/proc/cpuinfo`, but that won't work in general. One approach would be to check for `__AVX__` in the source files and just don't compile or register anything if it isn't defined. Otherwise we start to need configuration tests, like try to compile code that depends on `__AVX__` to decide whether to add these source files.

Reply:
I found a way to do this by interrogating '-march=native' in the makefile to see if it contains AVX support.